### PR TITLE
add ansh.hackclub.com

### PR DESCRIPTION
**Pull request by ansh3108**

### CODE REVIEW

**Review by grymmy — DISMISSED** [review]
I believe we try to keep this file in alphabetical order - please move this to the appropriate location

File: hackclub.com.yaml [hunk 1]
```diff
@@ -332,7 +332,11 @@ alumni:
 angelhacks:
   - ttl: 1
     type: CNAME
-    value: cname.vercel-dns.com.
+    value: cname.vercel-dns.com.  
+ansh:
+  - ttl: 1
+    type: CNAME
+    value: ansh3108.github.io/website/.
```
Comment (grymmy) [inline hackclub.com.yaml:339]: I believe this is not a valid CNAME entry

Comment (devramsean0) [reply hackclub.com.yaml:339]: [This guide](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site) should help you set a correct one up

File: dino.icu.yaml [hunk 2]
```diff
@@ -56,6 +56,11 @@ andrew: # by https://github.com/anddddrew
   ttl: 1
   type: CNAME
   value: nijmeh.xyz.
+  
+ansh:
+   ttl: 1
+    type: CNAME
+    value: ansh3108.github.io/website/.
```
Comment (xkrishguptaa) [inline dino.icu.yaml]: syntax error

```suggestion
   ttl: 1
   type: CNAME
   value: ansh3108.github.io/website/.
```

**Review by xkrishguptaa — APPROVED** [review]

**Review by devramsean0 — APPROVED** [review]
LGTM!
This now appears to be correct!

**Comment by devramsean0** [comment]
Merging Now!

**Pull request merged**